src/components/TestDbWrite/TestDbWrite.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { db } from '../../lib/firebase.js';
+
+function TestDbWrite() {
+  async function handleClick() {
+    try {
+      await db.collection('items').add({
+        name: 'test list item',
+        createdAt: Date.now(),
+      });
+    } catch (err) {
+      console.log(err);
+    }
+  }
```

Comment:
Very nice! Excellent choice of using date to help distinguish the new documents to make testing easier. 👏🏽